projects/plugins/jetpack/modules/search/instant-search/lib/hooks/use-photon.js

```diff
@@ -12,7 +12,7 @@ import { useEffect, useState } from 'react';
  * @returns {string} - Image URL without any query strings.
  */
 function stripQueryString( url ) {
-	return url.split( '?', 1 )[ 0 ];
+	return url && url.split( '?', 1 )[ 0 ];
```

Comment:
This changes the return type from `string` to `false|string`. Could we ensure that this won't cause any breakage and update the docstring above (line 12)?

Reply:
Good point, should be fixed now.